runtime/termscounter_test.go

```diff
@@ -0,0 +1,60 @@
+package runtime_test
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"go.etcd.io/gofail/runtime"
+)
+
+var __fp_ExampleString *runtime.Failpoint = runtime.NewFailpoint("runtime_test", "ExampleString")
+
+func TestTermsCounter(t *testing.T) {
+	testcases := []struct {
+		name string
+		desc string
+		run  int
+		want string
+	}{
+		{
+			name: "runtime_test/ExampleString",
+			desc: `10*sleep(10)->1*return("abc")`,
+			run:  12,
+			want: "11",
```

Comment:
If you run loop `12` times, how the result is `11`?

Reply:
this happens because we are building terms here by saying: 
`desc: `10*sleep(10)->1*return("abc")`,`
sleep exactly 10 times and then return 1 time.
Even if the function is called more than 11 times, no terms will be executed, because its controlled by internal counter: 
`func (mc *modCount) allow() bool {
	if mc.c > 0 {
		mc.c--
		return true
	}
	return false
}` 
That way we can check if the fail point is executed exactly 11 times and not the number of function calls.